assets/javascripts/discourse/connectors/user-summary-stat/sentiment.hbs

```diff
@@ -0,0 +1,16 @@
+{{#if
+  (and
+    this.siteSettings.ai_sentiment_enabled
+    @outletArgs.model.sentiment
+    this.showSentiment
+  )
+}}
+  <li class="user-summary-stat-outlet sentiment">
```

Comment:
The `user-summary-stat` plugin outlet has:
```
@connectorTagName="li"
```

So this would nest, `<li>` inside an `<li>`. Maybe we should use `<div>` here instead?

Reply:
That is not happening

![image](https://github.com/discourse/discourse-ai/assets/1385470/04421e1f-4c26-4a6b-b520-0579d0e33399)